### PR TITLE
Add support for PHP 8

### PR DESCRIPTION
**Pull request by lepiaf**
PR for issue https://github.com/php-translation/symfony-bundle/issues/423

### CODE REVIEW

**Comment by lepiaf** [comment]
```
composer update ${COMPOSER_FLAGS} --prefer-dist --prefer-stable --no-interaction

Loading composer repositories with package information

Updating dependencies

Your requirements could not be resolved to an installable set of packages.

  Problem 1

    - php-translation/symfony-storage[dev-master, 2.1.0, ..., 2.2.0] require php ^7.2 -> your php version (8.0.0) does not satisfy that requirement.

    - php-translation/symfony-storage 2.2.x-dev is an alias of php-translation/symfony-storage dev-master and thus requires it to be installed too.

    - Root composer.json requires php-translation/symfony-storage ^2.1 -> satisfiable by php-translation/symfony-storage[2.1.0, 2.2.0, 2.2.x-dev (alias of dev-master)].

```

I've opened a PR https://github.com/php-translation/symfony-storage/pull/51

File: phpstan-baseline.neon [hunk 1]
```diff
@@ -145,7 +145,5 @@ parameters:
 			count: 2
 			path: Twig/TranslationExtension.php
 
-		-
-			message: "#^Interface Translation\\\\Bundle\\\\Translator\\\\TranslatorInterface extends unknown interface Symfony\\\\Component\\\\Translation\\\\TranslatorInterface\\.$#"
-			count: 1
-			path: Translator/TranslatorInterface.php
+	excludes_analyse:
+		- Translator/TranslatorInterface.php
```
Comment (lepiaf) [inline phpstan-baseline.neon:149]: I don't know how to manage this error

```
% make phpstan
2 ↵ ✹
Note: Using configuration file /project/phpstan.neon.dist.
 49/49 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------
----------------------------------------------------------------------
  Line   Translator/TranslatorInterface.php
 ------
----------------------------------------------------------------------
  31     Interface Translation\Bundle\Translator\TranslatorInterface
extends
         unknown interface
Symfony\Component\Translation\TranslatorInterface.
         bulb Learn more at
https://phpstan.org/user-guide/discovering-symbols
 ------
----------------------------------------------------------------------

 --
-------------------------------------------------------------------------
     Error
 --
-------------------------------------------------------------------------
     Error message "Interface
     Translation\Bundle\Translator\TranslatorInterface extends unknown
     interface Symfony\Component\Translation\TranslatorInterface."
cannot be
     ignored, use excludes_analyse instead.
 --
-------------------------------------------------------------------------
```

**Comment by lepiaf** [comment]
I've also open a PR to https://github.com/php-translation/extractor/pull/157

**Comment by Ciloe** [comment]
Any advance on this PR ?

**Comment by Kocal** [comment]
Hi, is there any updates for this? 

Thanks!

**Comment by welcoMattic** [comment]
Hi, I'm sorry, I have not free time to work on PHP8 support PRs those days (major incident on a famous French hosting company) + coming deadline to finish https://github.com/symfony/symfony/pull/38475.
May be @bocharsky-bw or @rvanlaak are available to review those PRs?

**Comment by Kocal** [comment]
No worries, I know you are hard working for adding translations providers for Symfony. Keep up the good work! :muscle:

**Comment by bocharsky-bw** [comment]
Thank you for working on this and sorry for the slow replies!

Currently, this PR is blocked by https://github.com/php-translation/symfony-storage/pull/51 and waiting for it to be merged first

**Comment by Nyholm** [comment]
There are some CI jobs that fail. But they are not related to this PR. 

Thank you for your work.

**Pull request merged**